app-editors/vscode/vscode-1.56.0.ebuild

```diff
@@ -0,0 +1,74 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit desktop pax-utils xdg
+
+DESCRIPTION="Multiplatform Visual Studio Code from Microsoft"
+HOMEPAGE="https://code.visualstudio.com"
+SRC_URI="
+	amd64? ( https://update.code.visualstudio.com/${PV}/linux-x64/stable -> ${P}-amd64.tar.gz )
+	arm? ( https://update.code.visualstudio.com/${PV}/linux-armhf/stable -> ${P}-arm.tar.gz )
+	arm64? ( https://update.code.visualstudio.com/${PV}/linux-arm64/stable -> ${P}-arm64.tar.gz )
+"
+
+RESTRICT="mirror strip bindist"
+
+LICENSE="Microsoft-VSCode"
```

Comment:
I do this from this EBUILD ? Or does it need a different approach, like opening a bug ticket ?

Reply:
Just add a commit here to place it in licenses/, personally do that in its own commit before adding the ebuild.

See [devmanual](https://devmanual.gentoo.org/general-concepts/licenses/index.html#adding-new-licenses) for details.

Looking at it, I think this need to be added to EULA license_groups too, seems kind of nasty and there's even stuff about data collection.

Reply:
Thanks !

Yes there is data collection, and some of it cannot be disabled. I think it is a good idea to submit vscodium along vscode into Gentoo : the EBUILD should be very similar since vscodium is the source code compiled without telemetry nor Microsoft's licence. With a few features missing I switched back to vscode few months ago due to some issues, I do not remember which ones.